design/baremetal-operator/firmware-image-update.md

```diff
@@ -0,0 +1,195 @@
+# Firmware Image Update
+
+## Status
+
+provisional
+
+## Summary
+
+Provide the ability to get and update vendor firmware version.
+
+## Motivation
+
+It desired to update vendor firmware before deploying the OS image of the host.

```

Comment:
FYI, for someone without context this isn't as motivating as you possibly imagine it is :)